### DISCUSSION TITLE
Useful information for parents, tourists and locals

### DISCUSSION THREAD

**Discussion by j13m126** [organicmaps/organicmaps#1282] [Ideas]
I would like to propose three, I think easy to implement, changes for three different groups of people.

- make [changing tables](https://wiki.openstreetmap.org/wiki/DE:Key:changing_table) searchable and visible
	- it would be great if one could search for changing tables. 
	- also, it would be great to show it with an icon that a toilet or restaurant has a changing_table (just like the icon for toilets for disabled people).
	
- make [secondary wikipedia links](https://wiki.openstreetmap.org/wiki/Key:wikipedia) visible
	- I recently found out about secondary wikipedia links. Several statues and plaques in my city have it and it's a pity they are not visible, especially for tourists. 
	
- show [inscriptions](https://wiki.openstreetmap.org/wiki/Key:inscription) of plaques and statues. 
	- inscriptions are also limited to 255 characters, so it won't take too much space I guess.

**Comment by biodranik** [comment]
That is a very good idea! Please create an issue for that. Or even better, a separate issue for each idea :)

**Reply by j13m126** [reply]
Thank you, I've opened an issue for each of them. here they are: 

- [changing_table](https://github.com/organicmaps/organicmaps/issues/1301)
- [secondary wikipedia links](https://github.com/organicmaps/organicmaps/issues/1302) 
- [inscriptions](https://github.com/organicmaps/organicmaps/issues/1303)
